src/components/charts/AllMonthColumnChart.js

```diff
@@ -0,0 +1,33 @@
+import React, { Fragment, useMemo } from "react";
+import { ColumnChart, Model } from "@gooddata/react-components";
+
+// CONSTANTS
+import { CHART_PARAMS } from "../../constants/chartParams";
+
+const AllMonthColumnChart = () => {
+  const measures = useMemo(() => [
```

Comment:
`measures` should be passed in as a prop so that the component can be re-used to render charts for different measures.